README.md

```diff
@@ -1 +1,12 @@
-# protobufs
+# Instill AI  APIs
+
+This repository is the interface definitions of public Instill AI APIs that support both REST and gRPC protocols. We think this repository can provide a better understanding of Instill AI APIs by reading the original interface definitions to help you utilize them more efficiently. You can also use these definitions with open source tools to generate client libraries, documentation, and other artifacts.
+
+## Overview
+
+Instill AI APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST and RPC versions of the API, which can be accessed over different wire protocols
```

Comment:
```suggestion
Instill AI APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST (via [gRPC-Gateway](https://github.com/grpc-ecosystem/grpc-gateway)) and RPC versions of the API, which can be accessed over different wire protocols
```

Reply:
Fixed in this [commit](https://github.com/instill-ai/protobufs/pull/20/commits/3f02c41c242b906bd0a39c9669057970d2150c49)